mlflow/R/mlflow/.create-test-env.R

```diff
@@ -15,7 +15,11 @@ reticulate::conda_install("xgboost", envname = mlflow:::mlflow_conda_env_name())
 # Pin h2o to prevent version-mismatch between python and R
 reticulate::conda_install("h2o==3.30.1.3", envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
 
+# The default timeout value (60 seconds) can be insufficient for `spark_install` to complete
+options(timeout=60 * 60)
```

Comment:
This doesn't mean `spark_install` takes 1 hour. It usually completes within a minute but can be really slow sometimes.